### PR TITLE
chore(core): updated core actions to use response API

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

**Review by mrclay — APPROVED** [review]
Looks right, have not tested obviously

**Comment by jdalsem** [comment]
it is just a rewrite of the actions files...

**Comment by mrclay** [comment]
Yeah, just reminding us this is all untested so having a few more looks would be good.

**Comment by hypeJunction** [comment]
Or start adding tests

**Comment by mrclay** [comment]
> Or start adding tests

Happy to accept this nudge toward that.

**Comment by hypeJunction** [comment]
It's possible to do, and will help during transition to OO

**Review by hypeJunction — COMMENTED** [review]
I think you've missed some

File: actions/comment/save.php [hunk 1]
```diff
@@ -11,44 +11,42 @@
...
+	if (elgg_is_xhr()) {
+		// @todo move to its own view object/comment/content in 1.x
+		echo elgg_view('output/longtext', [
+			'value' => $comment->description,
+			'class' => 'elgg-inner',
+			'data-role' => 'comment-text',
+		]);
 	}
+	
+	system_message(elgg_echo('generic_comment:updated'));
```
Comment (jdalsem) [inline actions/comment/save.php]: Yep had that on my todo list in #11123... need to put the data also in the response

Comment (hypeJunction) [inline actions/comment/save.php]: Yes, you can just put plain HTML as the first argument in `elgg_ok_response()`. It's smart enough to handle both scalar and non-scalar values.

**Pull request merged**